### DISCUSSION TITLE
Position of Popover changed when upgrading from @reactour/tour 3.3.0 to 3.4.0

### DISCUSSION THREAD

**Discussion by amc514** [elrumordelaluz/reactour#562] [General]
After upgrading from 3.3.0 to 3.4.0 we observed new behavior for the positioning of the Popover.  We haven't changed anything besides doing the version upgrade as part of our normal dependency upgrade process.  We rely on the default positioning, which we were very happy with until this version.  Is there any easy way to get back the old behavior without pinning our version at 3.3.0?

**Comment by elrumordelaluz** [comment]
Hi @amc514 and thanks for pointing this out. In fact, this minor version improves the positioning calculation but could have some side-effect. Mind sharing a reduced case in a sandbox with the positions and steps you are using in order to check the behaviour between both versions? Otherwise a small video/screen-recording could be also useful. Thanks!

**Comment by amc514** [comment]
Hi @elrumordelaluz. Here's some screenshots that illustrate an example:

Version 3.3.0:
![master](https://user-images.githubusercontent.com/9088576/236558522-f14c39af-a06d-436e-b0c7-5c5a663dffbf.png)

Version 3.4.0:
![develop](https://user-images.githubusercontent.com/9088576/236558553-03b57df1-9f8f-4447-a45d-e5eb387eadb9.png)

Like I said before, we aren't doing anything to customize the positioning of the Popover because we were happy with the default behavior.

Happy to help make a code sandbox, do you have any instructions for doing so?

**Comment by amc514** [comment]
Hi @elrumordelaluz - here are two code sandboxes that demonstrate the differing behavior between the two versions:

v3.3.0 example:
https://codesandbox.io/s/popover-position-example-v3-3-0-ujd67c
https://ujd67c.csb.app/

v3.4.0 example:
https://codesandbox.io/s/popover-position-example-v3-4-0-pl53dh
https://pl53dh.csb.app/

In v3.3.0, the popover would appear below the highlighted element with some default padding.  In v3.4.0, the popover now appears to the left and without any padding.
